uber/models/attendee.py

```diff
@@ -671,42 +676,60 @@ def is_transferable(self):
             not self.has_role_somewhere
 
     @property
-    def paid_for_a_swag_shirt(self):
+    def paid_for_a_shirt(self):
         return self.amount_extra >= c.SHIRT_LEVEL
 
     @property
-    def volunteer_swag_shirt_eligible(self):
+    def volunteer_event_shirt_eligible(self):
         """
-        Returns: True if this attendee is eligible for a swag shirt *due to
-            their status as a volunteer or staff*. They may additionally be
-            eligible for a swag shirt for other reasons too.
+        Returns a truthy value if this attendee either automatically gets a
+        complementary event shirt for being staff OR if they've eligible for a
+        complementary event shirt if they end up working enough volunteer hours
         """
-
-        # Some events want to exclude staff badges from getting swag shirts
+        # Some events want to exclude staff badges from getting event shirts
         # (typically because they are getting staff uniform shirts instead).
         if self.badge_type == c.STAFF_BADGE:
             return c.STAFF_ELIGIBLE_FOR_SWAG_SHIRT
         else:
             return c.VOLUNTEER_RIBBON in self.ribbon_ints
 
     @property
-    def volunteer_swag_shirt_earned(self):
-        return self.volunteer_swag_shirt_eligible and (
+    def volunteer_event_shirt_earned(self):
+        return self.volunteer_event_shirt_eligible and (
             not self.takes_shifts or self.worked_hours >= 6)
 
     @property
-    def num_swag_shirts_owed(self):
-        swag_shirts = int(self.paid_for_a_swag_shirt)
-        volunteer_shirts = int(self.volunteer_swag_shirt_eligible)
-        return swag_shirts + volunteer_shirts
+    def replacement_staff_shirts(self):
+        """
+        Staffers can choose whether or not they want to swap out one of their
+        staff shirts for an event shirt.  By default and if the staffer opts
+        into this, we deduct 1 staff shirt from the staff shirt count and add 1
+        to the event shirt count.
+        """
+        has_replacement = self.gets_staff_shirt \
+            and self.second_shirt in [c.UNKNOWN, c.STAFF_AND_EVENT_SHIRT]
+        return 1 if has_replacement else 0
```

Comment:
As a minor style note, I tried about half a dozen different ways to get different version of this method to fit in the 79 character limit that ``flake8`` enforces, and this was the best I could come up with.  I'm open to other suggestions.

I personally find the single-line if/else version to be the most readable, so it's possible that this version, which looks the best to me, ends up looking uglier to other people than some of the options I rejected.

Reply:
I forgot to mention, I had a great conversation about `flake8` with @kitsuta and @earl7399 at MAGCon. We all agreed that 79 characters is workable, but we'd be willing to bump that up to 99 characters if there's a compelling reason.

The angst it's causing you may be reason enough!

That being said, the three versions of this that I would consider are:

# 1. As it is written
This is actually kinda nice, because it keeps the `second_shirt` clause on a single line, and starts the line with the logical operator. It's a little unsightly because we have to use a `\` to break the line.
```
        has_replacement = self.gets_staff_shirt \
            and self.second_shirt in [c.UNKNOWN, c.STAFF_AND_EVENT_SHIRT]
        return 1 if has_replacement else 0
```

# 2. Breaking in a different place
This is slightly cleaner, as it uses the brackets as a natural breakpoint. The downside to this version is that it splits the `second_shirt` clause across two lines.
```
        has_replacement = self.gets_staff_shirt and self.second_shirt in [
            c.UNKNOWN, c.STAFF_AND_EVENT_SHIRT]
        return 1 if has_replacement else 0
```

# 3. Splitting the conditional
This is probably the cleanest, but I had to use a shorter variable name for `has_replacement`
```
        is_replaced = self.second_shirt in [c.UNKNOWN, c.STAFF_AND_EVENT_SHIRT]
        return 1 if self.gets_staff_shirt and is_replaced else 0
```

Reply:
I would definitely prefer a 99 character limit to a 79 character one!

In the meantime, I liked your 3rd suggestion so I've updated this method to use that approach.